getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -183,16 +183,29 @@ You can enable GitHub discoverability for your components in {product-short}, su
 
 .Procedure
 
-. In Red Hat Openshift, go to the *Helm* tab and upgrade the _Developer Hub_ release.
-. Under *global → Dynamic plugins configuration → List of dynamic plugins that should be installed in the backstage application. → Package specification of the dynamic plugin to install. It should be usable by the `npm pack` command.*, add the following value:
+. In the *Developer* perspective of the OpenShift web console, go to the *Helm* tab.
+. Click the three dots corresponding to a Helm release and select *Upgrade*.
+. Use one of the following views to edit the Helm configuration:
+
+** Using *Form view* 
++
+.. Expand *Root Schema → global → Dynamic plugins configuration → List of dynamic plugins that should be installed in the backstage application*. 
+.. Click the *Add List of dynamic plugins that should be installed in the backstage application* link.
+.. In the *Package specification of the dynamic plugin to install. It should be usable by the `npm pack` command.* field, add the following value:
 +
 --
 `./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic`
 
 image::rhdh/enable-gh-discovery.png[]
 --
 
-. Click *Upgrade*.
+.. Click *Upgrade*.
+
+** Using *YAML view* 
++
+.. Set the value of the `global.dynamic.plugins.package` parameter to `./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic`.
```

Comment:
Same here, it is better visualizing how the YAML manifest is represented. For example:

```yaml
global:
  dynamic:
    # other Red Hat Developer Hub Helm Chart configuration
    plugins:
      - disabled: false
        package: >-
          ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic
    # other Red Hat Developer Hub Helm Chart configuration
```